pkgs/development/python-modules/clickhouse-cli/default.nix

```diff
@@ -10,11 +10,12 @@
 
 buildPythonPackage rec {
   pname = "clickhouse-cli";
-  version = "0.3.8";
+  version = "0.3.9";
+  format = "pyproject";
```

Comment:
It no longer works:

```
$ clickhouse-cli
Traceback (most recent call last):
  File "/Users/ivan/.nix-profile/lib/python3.11/importlib/metadata/__init__.py", line 563, in from_name
    return next(cls.discover(name=name))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ivan/projects/clickhouse-cli-venv/venv/bin/clickhouse-cli", line 33, in <module>
    sys.exit(load_entry_point('clickhouse-cli==0.3.8', 'console_scripts', 'clickhouse-cli')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ivan/projects/clickhouse-cli-venv/venv/bin/clickhouse-cli", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
                       ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ivan/.nix-profile/lib/python3.11/importlib/metadata/__init__.py", line 981, in distribution
    return Distribution.from_name(distribution_name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/ivan/.nix-profile/lib/python3.11/importlib/metadata/__init__.py", line 565, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: No package metadata was found for clickhouse-cli
```

It also didn't work before due to sqlalchemy (https://github.com/hatarist/clickhouse-cli/issues/50), but it failed later.

Reply:
Thanks for your report.
Unfortunately, we cannot check that all packages are working correctly.
Would you please either fix the packages or if necessary, revert and submit a PR? Feel free to ping me on that.
Also, I would appreciate it if you would consider adding a test, which will help us notice the breakage more quickly.

Reply:
I'm using some overlay as workaournd

```nix
{ channels, ... }:

final: prev: {
  clickhouse-cli = channels.unstable.python311Packages.clickhouse-cli.override {
    sqlparse = channels.unstable.python311Packages.sqlparse.overridePythonAttrs (oldAttrs: {
      src = prev.fetchPypi {
        pname = "sqlparse";
        version = "0.4.3";
        hash = "sha256-acqASEa7EU0uw4DkNgqKNA24PwzPOvzusUBN8Cj1cmg=";
      };
      nativeBuildInputs = [
        prev.python311Packages.flit-core
        prev.python311Packages.setuptools
        prev.installShellFiles
      ];
    });
  };
}
```